firebase.json

```diff
@@ -5,16 +5,14 @@
   },
   "hosting": {
     "public": "public",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],  
-    "rewrites": [{
-      "source": "**",
-      "function": "api",
-      "region": "us-central1"
-    }],
+    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
+    "rewrites": [
+      {
+        "source": "**",
+        "function": "api",
+        "region": "us-central1"
+      }
+    ],
```

Comment:
nit: revert these changes since you're not changing the functionality and to reduce the number of diffs